src/carousel.ts

```diff
@@ -22,21 +26,25 @@ function carousel(id: string, curChild=0) {
         centerCard(curChild);
     });
 
-    divLeft.addEventListener('click', () => {
-        if (curChild === 0) {
-            return;
-        }
-        centerCard(--curChild);
-        showControls();
-    });
 
-    divRight.addEventListener('click', () => {
-        if (curChild === numChildren - 1) {
-            return;
+    //Enable Swipe on mobile/tablet devices only
+    hammer.set({enable: windowSize <= 1024 ? true : false});
```

Comment:
Why not get rid of this altogether?  And aren't some iPad dimensions over 2000 pixels?  Not to mention other devices with touch screens.  This doesn't hurt to have this library loaded on all devices, does it?

Reply:
It actually works quite well on desktop by clicking and dragging. Only issue is if a user tries to click and drag on the image it drags a ghost image and it doesn't swipe. I believe the ghost image issue should be an easy fix though and can create another issue for that and handle.